### PR TITLE
Mails

### PR DESCRIPTION
**Pull request by TheRittler**
implementation of mail package

### CODE REVIEW

File: src/test/java/edu/kit/ipd/crowdcontrol/objectservice/mail/GmailMailTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        props.put("mail.smtp.auth", "true");
+        props.put("mail.smtp.starttls.enable", "true");
+        props.put("mail.smtp.tls", "true");
+        props.put("mail.smtp.ssl.checkserveridentity", "true");
+        props.put("mail.store.protocol", "imap");
+        props.put("mail.imap.host", "imap.gmail.com");
+        props.put("mail.imap.port", "993");
+        props.put("mail.imap.ssl", "true");
+        props.put("mail.imap.ssl.enable", "true");
+        java.security.Security.addProvider(new com.sun.net.ssl.internal.ssl.Provider());
```
Comment (marcelhollerbach) [inline src/test/java/edu/kit/ipd/crowdcontrol/objectservice/mail/GmailMailTest.java]: Can you import java.security.Securits and com.sun.net.ssl.internal.ssl.Provider ? Just for the visual fun.

Comment (marcelhollerbach) [inline src/test/java/edu/kit/ipd/crowdcontrol/objectservice/mail/GmailMailTest.java]: Well thats just a test .... nvm.

**Comment by kelunik** [comment]
Please move these integration tests to `/src/integration-test/...`.

**Comment by LeanderK** [comment]
in the future please use rebase instead of merging the master into you branch

**Comment by kelunik** [comment]
Rebase would require force pushes for branches that are already pushed.

**Comment by TheRittler** [comment]
Can we merge and close here?

**Comment by kelunik** [comment]
Repository should still not contain credentials to (publicly accessible) mail boxes.

**Pull request merged**